packaging/addons/openproject/bin/postinstall

```diff
@@ -12,7 +12,7 @@ ${CLI} config:set RECOMPILE_RAILS_ASSETS="true"
 
 # Set the configured default language
 # Will be unset at installation postinstall before restart to ensure the setting is writable
-${CLI} config:set OPENPROJECT_DEFAULT_LANGUAGE="$(wiz_get "openproject/default_language")"
+${CLI} config:set OPENPROJECT_DEFAULT_LANGUAGE="$(wiz_get "openproject/default_language" || echo "en")"
```

Comment:
OpenProject starts to parse the value and as it's empty, it will fail. We could either wrap the check here to avoid setting an env, or keep the `en` fallback. The latter is fine for me.